metricbeat/modules.d/aerospike.yml.disabled

```diff
@@ -1,5 +1,5 @@
 # Module: aerospike
-# Docs: https://www.elastic.co/guide/en/beats/metricbeat/6.x/metricbeat-module-aerospike.html
+# Docs: https://www.elastic.co/guide/en/beats/metricbeat/6.6/metricbeat-module-aerospike.html
```

Comment:
Hmm, for the next minor, we'll need to merge this earlier, so it makes it into the actual release.

Reply:
Yep, just noticed it when it was too late :(

Next time we will do the version bump when we create a release branch.